app/views/layouts/application.html.erb

```diff
@@ -20,8 +20,10 @@
 
   
   <body>
+
     <%= render partial: "shared/navbar" %>
     <div class="container main-content ">
+      <%# <%= render partial: "shared/alert" %>
```

Comment:
this line is not running because its been commented out. make sure to uncomment it

Reply:
deleted and may fix in future.